Patterns/Swedish/Swedish-TimeZone.yaml

```diff
@@ -314,7 +611,7 @@ FullNameList: !list
     - Pacific
     - Eastern
 BaseTimeZoneSuffixRegex: !simpleRegex
-  def: ((\s+|-)(friendly|compatible))?(\s+|-)time(zone)?
+  def: ((\s+|-)(friendly|compatible))?(\s+|-)(ti(d|me))(zone)?
```

Comment:
Is there a coloquial expression similar to "friendly" or "compatible", as in the example in spec that would be used in Swedish between the location/timezone and the time/timezone word?

Reply:
Not that I can think of.